vernac/classes.ml

```diff
@@ -69,17 +69,17 @@ let add_instance_hint inst path ~locality info =
 (* short names without opening all Hints *)
 type locality = Hints.hint_locality = Local | Export | SuperGlobal
 
-type instance_obj = {
-  inst_class : GlobRef.t;
-  inst_info: hint_info;
+type instance = {
+  class_name : GlobRef.t;
```

Comment:
Not a big deal at all, but I find that wrapping types in modules is precisely most useful to scope record fields, so indeed the suggestion would have been `Instance.t` too, but indeed that just styling.